tests/Integration/Behaviour/Features/Scenario/Meta/meta_management.feature

```diff
@@ -0,0 +1,150 @@
+    Given I specify following properties for new meta "meta10":
+      | meta_id                            | 4                                |
+      | localized_rewrite_urls             | i-got-char-š                     |
+    And shop configuration for "PS_ALLOW_ACCENTED_CHARS_URL" is set to 0
+    When I update meta "meta10" with specified properties
+    Then I should get error that url rewrite value is incorrect
+
+  Scenario: Updating metadata with invalid url rewrite should be allowed when ascended chars setting is turned on
+    Given I specify following properties for new meta "meta11":
+      | meta_id                            | 4                                |
+      | localized_rewrite_urls             | i-got-char-š                     |
+    And shop configuration for "PS_ALLOW_ACCENTED_CHARS_URL" is set to 1
+    When I update meta "meta11" with specified properties
+    Then meta "meta11" page should be "index"
+    And meta "meta11" field "url_rewrite" for default language should be "i-got-char-š"
+
+  Scenario: Updating metadata with non existing page name
+    Given I specify following properties for new meta "meta11":
+      | meta_id                            | 4                                |
+      | page_name                          | i-dont-exist                     |
+    When I update meta "meta11" with specified properties
+    Then I should get error that page name value is incorrect
+
+  Scenario: Updating metadata with already existing page name
+    Given I specify following properties for new meta "meta12":
+      | meta_id                            | 4                                |
+      | page_name                          |  	pagenotfound                  |
+    When I update meta "meta11" with specified properties
+    Then I should get error that page name value is incorrect
+
+  Scenario: Get non existing meta for editing it raises an error
+    Given I specify following properties for new meta "meta12":
+      | meta_id                            | 9999                             |
+    When I get meta "meta12" with specified properties
+    Then I should get error that meta entity is not found
+
+  Scenario: Get existing meta for editing
+    Given I specify following properties for new meta "meta13":
+      | meta_id                          | 4                                     |
+      | page_name                        | index                                 |
+      | localized_page_title             | page title in default language        |
+      | localized_meta_description       | meta description in default language  |
+      | localized_meta_keywords          | meta keywords in default language     |
+      | localized_rewrite_urls           | rewrite-url-default                   |
+    When I update meta "meta13" with specified properties
+    When I get meta "meta13" with specified properties
+    Then meta "meta13" editable form field "page" should be equal to "index"
```

Comment:
I'm wondering if we should try to make these steps "the command saved everything successfully' automatic, so we can apply this to every add/edit command.

A step "everything was saved successfully" with a bit of magic in it

Maybe a bad idea

I dont know

Tell me 😅

Reply:
the command and the object has different fields associated - Im afraid not unless we specify some kind of mapping but it would be too much for such task I guess :smile: